libs/api/domains/social-insurance/src/lib/dtos/temporaryCalculation.input.ts

```diff
@@ -0,0 +1,70 @@
+import { InputType, Field, Int } from '@nestjs/graphql'
+
+@InputType('SocialInsuranceIncomeType')
+class IncomeType {
+  @Field(() => Int, { nullable: true })
+  incomeTypeNumber?: number
+
+  @Field()
+  incomeTypeCode?: string
```

Comment:
Is this nullable or not? The decorator says required, but the property says nullable.
Might want to review the others as well